src/sentry/monitors/models.py

```diff
@@ -74,14 +74,22 @@ class MonitorEnvironmentValidationFailed(Exception):
     pass
 
 
+class MonitorObjectStatus(ObjectStatus):
+    WAITING = 4
```

Comment:
might want to use a different value to make sure this doesn't get overwritten or just completely redefine the `ObjectStatus` to avoid the issue